Neos.Flow/Classes/ObjectManagement/Configuration/ConfigurationBuilder.php

```diff
@@ -507,7 +507,11 @@ protected function autowireProperties(array &$objectConfigurations)
                 continue;
             }
 
-            $classMethodNames = get_class_methods($className);
+            try {
+                $classMethodNames = get_class_methods($className);
+            } catch (\TypeError $error) {
+                throw new UnknownClassException(sprintf('The class "%s" defined in the object configuration for object "%s", defined in package: %s, does not exist.', $className, $objectConfiguration->getObjectName(), $objectConfiguration->getPackageKey()), 1352371371);
```

Comment:
This should get a new exceptipn code…

Reply:
Hm, it is the same cause, just in PHP8-case - but maybe it's not breaking to have a slightly different exception code? Not 100% sure
```suggestion
                throw new UnknownClassException(sprintf('The class "%s" defined in the object configuration for object "%s", defined in package: %s, does not exist.', $className, $objectConfiguration->getObjectName(), $objectConfiguration->getPackageKey()), 1352371372);
```